hummingbot/connector/connector/uniswap/uniswap_connector.py

```diff
@@ -48,6 +48,7 @@ class UniswapConnector(ConnectorBase):
     """
     API_CALL_TIMEOUT = 10.0
     POLL_INTERVAL = 60.0
+    UPDATE_ORDER_STATUS_MIN_INTERVAL = 10.0
```

Comment:
same, we don't need this.

Reply:
Changed to set balance polling interval.